mora/util.py

```diff
@@ -97,10 +99,16 @@ def parsedatetime(s: str) -> datetime.datetime:
     try:
         dt = dateutil.parser.parse(s, dayfirst=True, tzinfos=_tzinfos)
     except ValueError:
-        raise exceptions.HTTPException(
-            exceptions.ErrorCodes.E_INVALID_INPUT,
-            'cannot parse {!r}'.format(s)
-        )
+        if default is not _sentinel:
+            return default
+        else:
+            raise exceptions.HTTPException(
+                exceptions.ErrorCodes.E_INVALID_INPUT,
+                'cannot parse {!r}'.format(s)
+            )
+
+    if dt.date() == POSITIVE_INFINITY.date():
```

Comment:
This seems like a change specific to how the data is for AAK.
I'd rather see us have the specific handling in the individual importers, than have a number of special cases here.

Reply:
I see. I guess this is a general enough handling to be included in here.